Changelog.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added `VK_NV_memory_decompression` device extension (#761)
 - Added `Handle::is_null()` to allow checking if a handle is a `NULL` value (#694)
 - Allow building `Entry`/`Instance`/`Device` from handle+fns (see their `from_parts_1_x()` associated functions) (#748)
 - Update Vulkan-Headers to 1.3.254 (#760)
```

Comment:
Let's keep this sorted chronologically.

```suggestion
- Added `Handle::is_null()` to allow checking if a handle is a `NULL` value (#694)
- Allow building `Entry`/`Instance`/`Device` from handle+fns (see their `from_parts_1_x()` associated functions) (#748)
- Update Vulkan-Headers to 1.3.254 (#760)
- Added `VK_NV_memory_decompression` device extension (#761)
```

Reply:
Hmm I cannot push to your branch nor apply this requested change before merging, so will have to wait for @BeastLe9enD to apply it.